docs/docs/deploy/google-cloud-run.md

```diff
@@ -61,6 +61,8 @@ Running this command will do three things:
 - Build the Docker image in [Cloud Build](https://cloud.google.com/build) and upload it to [Artifact Registry](https://cloud.google.com/artifact-registry)
 - Deploy the image to the specified Cloud Run service
 
+There is an ongoing [issue](https://github.com/google/gvisor/issues/7331) that is preventing Dart Unix Sockets from working correctly in the gen1 version of the Cloud Run environment. In that case, it is recommended to deploy to gen2 by adding `--execution-environment=gen2`
```

Comment:
Aside from fixing the issue you mentioned, does switching to gen2 do anything else/extra that we should make the user aware off?

Reply:
Nothing that im aware off tbh. Was looking around, but the socket issue was only thing that i found

Reply:
@wolfenrain is it ok for you?